torch/csrc/jit/codegen/cuda/ir_utils.h

```diff
@@ -282,6 +282,12 @@ TORCH_CUDA_CU_API std::vector<TensorView*> outputTvsOf(
 // returns all tensor views in fusion that are used between outputs and inputs.
 TORCH_CUDA_CU_API std::vector<TensorView*> allTvs(Fusion* fusion);
 
+// returns all tensor views in fusion that are used between outputs and inputs
+// except the specified set.
+TORCH_CUDA_CU_API std::unordered_set<TensorView*> allTvsExcept(
```

Comment:
Might want to return a vector for reproducibility

Reply:
fixed